### PR TITLE
Fixed cyrrilic problem

### PR DESCRIPTION
**Pull request by VitalijF**

### CODE REVIEW

File: src/main/java/ua/softserveinc/tc/controller/user/BookingTimeController.java [hunk 1]
```diff
@@ -59,7 +54,8 @@ public String getRoomProperty(@RequestBody Integer roomId) {
         return null;
     }
 
-    @PostMapping("makenewbooking")
+    @PostMapping(value ="makenewbooking", produces = "application/json; charset=UTF-8")
+    @ResponseBody
```
Comment (hrybs) [inline src/main/java/ua/softserveinc/tc/controller/user/BookingTimeController.java]: This is RestController this line is needless

Comment (VitalijF) [inline src/main/java/ua/softserveinc/tc/controller/user/BookingTimeController.java]: Response body?

Comment (hrybs) [inline src/main/java/ua/softserveinc/tc/controller/user/BookingTimeController.java]: Yes. This is a RestController, so there is implicit ResponseBody. You needn't explicitly write it.

Comment (VitalijF) [inline src/main/java/ua/softserveinc/tc/controller/user/BookingTimeController.java]: Fixed in new commit

**Pull request merged**